explore/src/main/scala/explore/ExploreLayout.scala

```diff
@@ -172,15 +172,16 @@ object ExploreLayout:
 
           given Display[AppTab] = _.title
 
-          val (showProgsPopup, msg, isSubmitted) =
-            props.view.get.programSummaries.fold((false, none, false)) { pss =>
-              routingInfo.optProgramId.fold((true, none, false)) { id =>
+          val (showProgsPopup, msg, isSubmitted, ref) =
```

Comment:
A different name than `ref` might be nice. Maybe `proposalRef`? When I read it I thought it referred to a react ref.

Reply:
👍